low-level/sensors/impl/generic/node_hw.py

```diff
@@ -908,15 +908,17 @@ def _parse_fan_info(self, index, date, _time, device_id, sensor_id,\
                              "Upper Critical", "Fully Redundant", "State Asserted",
                              "State Deasserted"]
 
-        threshold_event = event.split(" ")
-        threshold = threshold_event[len(threshold_event)-1]
-
         if status.lower() == "deasserted" and event.lower() == "fully redundant":
             alert_type = "fault"
         elif status.lower() == "asserted" and event.lower() == "fully redundant":
             alert_type = "fault_resolved"
-        elif threshold.lower() in ['low', 'high']:
-            alert_type = f"threshold_breached:{threshold}"
+        elif "going high" in event.lower() or "going low" in event.lower():
```

Comment:
I think you can still use old logoc:
1. split the string (old code: line no 911)
2. Get the changed state: (old code: line no 911)
3. Check whether threshold state matches in the list (old code: line no 918)
4. Check assert/deassert then and populate the alert_type.

Please let me know if you find some problem with this implementation. 
One more thing which I also missed it. Its just a suggestion from me. Many times when we read string, it comes with extra chars like new line etc. So, before you use it, just strip it.

Reply:
I don't think there is any problem with splitting and reading the last word in the event string (i.e. "high" or "low"), I just made this change because this one is a shorter way of doing the same thing. If "going" is not required, we can also have
`elif "high" in event.lower() or "low" in event.lower():`
But I think we can keep "going" for safety.

About stripping spaces, that is a good suggestion... I will add those changes.

Reply:
Am ok with either case